src/style.css

```
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--1 :nth-child(1) {
```

```suggestion
.stars--1 :nth-child(1),
.stars--2 :nth-child(-n + 2),
... ,
... {
```
Consider using commas to group selectors and avoid code-repeating